### ISSUE TITLE
Where is the ExampleFuzzerWithNativeASan and ExampleFuzzerWithNativeUBSan?

### ISSUE THREAD

**Issue by liuyangc3** [CodeIntelligenceTesting/jazzer#645]
From https://github.com/CodeIntelligenceTesting/jazzer/blob/main/docs/advanced.md
> The fuzz targets `ExampleFuzzerWithNativeASan` and `ExampleFuzzerWithNativeUBSan` in the [examples](https://github.com/CodeIntelligenceTesting/jazzer/blob/main/examples/src/main/java/com/example) directory contain minimal working examples for fuzzing with native libraries. Also see TurboJpegFuzzer for a real-world example.

I didn't find those two examples in the dir.

**Comment by liuyangc3** [comment]
```bash
cd example
bazel build //:ExampleFuzzerWithASan
                            
ERROR: Skipping '//:ExampleFuzzerWithASan': no such target '//:ExampleFuzzerWithASan': target 'ExampleFuzzerWithASan' not declared in package '' defined by /Users/steven/workspace/blockchain-sec/jazzer/BUILD.bazel (Tip: use `query "//:*"` to see all the targets in that package)
WARNING: Target pattern parsing failed.
ERROR: no such target '//:ExampleFuzzerWithASan': target 'ExampleFuzzerWithASan' not declared in package '' defined by /Users/steven/workspace/blockchain-sec/jazzer/BUILD.bazel (Tip: use `query "//:*"` to see all the targets in that package)
INFO: Elapsed time: 0.108s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
```

**Comment by kyakdan** [comment]
Thanks for reporting this issue. The names of these examples have changed. You can run example `bazel run //examples:ExampleFuzzerWithASan`. Does this work for you?

I'll fix the documentation with the adjusted naming.

**Comment by kyakdan** [comment]
Naming is fixed here #646

**Comment by liuyangc3** [comment]
> Thanks for reporting this issue. The names of these examples have changed. You can run example `bazel run //examples:ExampleFuzzerWithASan`. Does this work for you?
> 
> I'll fix the documentation with the adjusted naming.

Yes bazel run //examples:ExampleFuzzerWithASan can work thanks
